lib/utils.js

```diff
@@ -38,27 +38,44 @@ exports.makeGetStationResponseText = function (trainArrivals, stationName) {
     responseText = props.stationNoArrivalsSpeechOutput;
   } else if (_.size(trainArrivals) === 1) {
     var destinationStationName = _.keys(trainArrivals)[0];
-    responseText = 'The next train to ' + destinationStationName + ' arrives in ' +
+    responseText = this.replaceAbbreviations('The next train to ' + destinationStationName + ' arrives in ' +
       trainArrivals[destinationStationName] + ' minutes. There are no other trains arriving at ' + stationName +
-      ' in the next 20 minutes.';
+      ' in the next 20 minutes.', abbreviations['arrivals']);
   } else {
-    responseText = 'Are you going to ' + this.joinListConjuction(_.keys(trainArrivals), ', ', ' or ') + '?';
+    responseText = 'Are you going to ' + this.joinListConjunction(_.keys(trainArrivals), ', ', ' or ') + '?';
   }
   return responseText;
 };
 
-exports.sanitizeServiceAdvisories = function (serviceAdvisoryText) {
-  var replacements = {
-    btwn: 'between',
-    Blu: 'Blue',
-    Org: 'Orange'
-  };
+exports.makeGetDestinationResponseText = function (arrivalTimes, destinationStationName) {
+  var brdOrArr = _.remove(arrivalTimes, function (arrivalTime) {
+    return arrivalTime === 'BRD' || arrivalTime === 'ARR';
+  });
+  var responseText = '';
+  if (_.size(brdOrArr) > 0) {
+    // in the unlikely even that there are back to back BRD and ARR, just tell them about BRD. they won't miss anything
+    responseText += this.replaceAbbreviations('The next train to ' + destinationStationName + ' is ' + brdOrArr[0] +
+      ' now.', abbreviations['arrivals']);
+    if (_.size(arrivalTimes) > 0) {
```

Comment:
I think the logic is bad here, and it is actually complicating things, so once it is fixed, all these Ifs simplify, because the response text can be reused for various cases.

Think about if a train is arriving at ballston in the direction of vienna, a second train is 5 minute from ballston in the direction of largo, and the third train is 6 minutes from ballston in the directon of vienna. The code would say:
"The next train to Vienna is arriving now. There are also trains arriving in five and six minutes." But you don't know which direction those trains are. And if nothing was arriving, you would know, because you say the direction.

After fixing that logic, you can reuse a lot of code, and then you just need to insert the word "Also," if a train is arriving and there are more trains. I think this will be much cleaner.